hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/DeletedBlockLogImpl.java

```diff
@@ -425,7 +425,11 @@ public DatanodeDeletedBlockTransactions getTransactions(
           DeletedBlocksTransaction txn = keyValue.getValue();
           final ContainerID id = ContainerID.valueOf(txn.getContainerID());
           try {
-            if (txn.getCount() > -1 && txn.getCount() <= maxRetry
+            if (containerManager.getContainer(id).isDeleted()) {
+              LOG.warn("Container: " + id + " was not deleted for the " +
```

Comment:
There seems to be a typo in the log. I believe it should be was deleted instead.